runtime/src/comm/ofi/comm-ofi-launch.c

```diff
@@ -84,4 +84,9 @@ void chpl_comm_preLaunch(int32_t numLocales) {
     //
     chpl_env_set("PMI_NO_PREINITIALIZE", "y", 1);
   }
+  //
+  // Don't map virtual hugepages to physical pages if we allocate a
+  // fixed heap.
+  //
+  chpl_env_set("HUGETLB_NO_RESERVE", "yes", 0);
```

Comment:
I wonder if we should only set this if a user has requested hugepage support. It _probably_ won't hurt anything to always set it, but I'd feel a little better if we limited when we set it to start.

Reply:
Good point, I'll make the change.